assets/component-facets.css

```diff
@@ -744,20 +810,20 @@ details.menu-opening .mobile-facets__close svg {
 }
 
 .mobile-facets__highlight {
-  opacity: 0;
-  visibility: hidden;
-}
-
-.mobile-facets__checkbox:checked + .mobile-facets__highlight {
-  visibility: visible;
-  opacity: 1;
   position: absolute;
   top: 0px;
   left: 0px;
   right: 0px;
   bottom: 0px;
   display: block;
   background-color: rgba(var(--color-foreground), 0.04);
+  opacity: 0;
+  visibility: hidden;
+}
+
+.mobile-facets__checkbox:checked + .mobile-facets__highlight {
+  opacity: 1;
+  visibility: visible;
 }
 
```

Comment:
Is there a reason why certain colors show the opacity different when unavailable while some have the `/`?

https://screenshot.click/12-06-17qnp-wwrsu.png

Reply:
We use the current foreground color at 40% opacity, along with setting the opacity of the image itself to 40%. Perhaps we could boost the opacity to make it more apparent, but the opacity itself also signifies it's disabled. Any suggestions?

Reply:
I was noticing this too and think it's something we should fix. Is there a way we can just not apply the decreased opacity to the /?